docs/core/porting/winforms.md

```diff
@@ -31,7 +31,9 @@ In this article, various names are used to identify types of files used for migr
 - [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) 3.0 or later.
 
 > [!NOTE]
-> **Visual Studio 2017** doesn't support .NET Core 3.0 projects. **Visual Studio 2019** supports .NET Core 3.0 projects but doesn't yet support the visual designer for .NET Core 3.0 Windows Forms projects. To use the visual designer, you must have a .NET Windows Forms project in a solution that shares the forms files with the .NET Core project.
+> **Visual Studio 2017** doesn't support .NET Core 3.0 projects. **Visual Studio 2019** supports .NET Core 3.0 projects. Starting with **Visual Studio 2019 version 16.5 Preview 1**, the .NET Core Windows Forms designer is also supported.
+>
+> To enable the designer, Go to **Tools** > **Options** > **Environment** > **Preview Features** and select the **Use the preview Windows Forms designer for .NET Core apps** option.
 
 ### Consider
 
```

Comment:
probably line 58 should also be changed. and ms.date updated since VS 2019 16.5 was launched after March 2019.